diff --git a/dist/shared/changelogen.a79f5d5c.mjs b/dist/shared/changelogen.a79f5d5c.mjs
index 901f6e360af4b6ea62758ebe28f0758d306a41f0..e721168371ed7521fabd4fefa881f06ee3a83725 100644
--- a/dist/shared/changelogen.a79f5d5c.mjs
+++ b/dist/shared/changelogen.a79f5d5c.mjs
@@ -338,12 +338,8 @@ async function generateMarkDown(commits, config) {
       "### \u2764\uFE0F Contributors",
       "",
       ...authors.map((i) => {
-        const _email = [...i.email].find(
-          (e) => !e.includes("noreply.github.com")
-        );
-        const email = _email ? `<${_email}>` : "";
-        const github = i.github ? `([@${i.github}](http://github.com/${i.github}))` : "";
-        return `- ${i.name} ${github || email}`;
+        const github = i.github ? ` ([@${i.github}](http://github.com/${i.github}))` : "";
+        return `- ${i.name}${github}`;
       })
     );
   }
